src/__tests__/__snapshots__/index.spec.ts.snap

```diff
@@ -310,22 +310,22 @@ Object {
   "steps": Array [
     Object {
       "command": "web/bin/buildkite/run_web_step.sh build editor",
-      "key": "_0",
+      "key": "_5",
```

Comment:
@joscha The TS v4.0 upgrade seems to have changed how `uniqid()` returns values (or in what order steps are processed and call `uniqid()`?). The keys all still line up fine, but this may be a breaking change.

Reply:
this is most likely due to moving the retry/env parts up, which means they get earlier keys and later ones shift. Not a breaking change as these IDs are just guaranteed to be unique, not stable.